frontend/public/components/app.jsx

```diff
@@ -144,6 +144,7 @@ class App_ extends React.PureComponent {
     const { isNavOpen, isDrawerInline } = this.state;
     const { contextProviderExtensions } = this.props;
     const { productName } = getBrandingDetails();
+    const PageSkipToContent = <SkipToContent href={'#content'}>Skip to Content</SkipToContent>;
```

Comment:
Hi @jessiehuff, can you pull this outside of render so it's a separate top-level component? If it's an anonymous component here, it will unmount/remount every render of `App`.